params: v1.9.0

meowsbits opened the pull request.

meowsbits committed:
params: v1.9.0

whilei commented on params/version.go:
```diff
@@ -21,10 +21,10 @@ import (
+	VersionMajor = 1        // Major version component of the current release
+	VersionMinor = 9        // Minor version component of the current release
+	VersionPatch = 0        // Patch version component of the current release
+	VersionMeta  = "tagged" // Version metadata to append to the version string
```
```suggestion
	VersionMeta  = "unstoppable" // Version metadata to append to the version string
```

meowsbits replied:
@tzdybal @sorpaas But actually, here, suggesting we can use improved descriptors over "stable" and "unstable"? 

Specifically, we want to say "on tag" (which would correspond to published release) or off-tag, which would correspond to saying built-from-master.

Since it's either one or the other, we only need to describe one case and the other will be the exclusive default.

~~Or... we could use a new pattern where the `VersionMeta` value would be derived from, say, the branch name (by the same processes at the `gitCommit` value in use). This isn't perfect, and it can yield "false-positives," but in most cases I'd expect it to be conveniently descriptive.~~

meowsbits committed:
params: use 'release' versionmeta value

... instead of 'un/stable' or 'tagged'.

This seems the simplest and most self-explanatory.

tzdybal reviewed (approved):
LGTM
